ci/install.sh

```diff
@@ -89,7 +89,7 @@ if [ ! -e "${GPG21_INSTALL}/bin/gpg" ]; then
     cd ..
   done
 
-  for archive in libgcrypt:1.8.0 libassuan:2.4.3 libksba:1.3.5; do
+  for archive in libgcrypt:1.8.2 libassuan:2.5.1 libksba:1.3.5; do
```

Comment:
Could we find some way in not hardcoding the package versions but dynamically determine them based on GnuPG version? I find this very difficult to manage...

Reply:
I agree, but this PR is just quick fix rather than big improvement of those scripts (which probably is needed anyway).

Reply:
Agree, let's just merge this first and leave generalizing to a new issue.